knative-kubecon/README.md

```diff
@@ -329,25 +326,26 @@ spec:
       name: helloworld-openshift
 ```
 
-This _Subscription_ again references the `testchannel` and also defines a `Subscriber`, in this case the Serving application `helloworld-openshift` we built earlier.
+This Subscription again references the `testchannel` and also defines a `Subscriber`, in this case the Serving application `helloworld-openshift` we built earlier.
 This basically means that we have subscribed the previously built application to the `testchannel`.
 Once we apply the `Subscription` any Kubernetes platform events from the `myproject` namespace will be routed to the `helloworld-openshift` application.
 Behind the scenes there is a `SubscriptionController` which is doing the wiring for us.
 
 ```bash
-$ oc apply -f 030-subscription.yaml -n myproject
+oc apply -f eventing/030-subscription.yaml -n myproject
 ```
 
-If we take a look at the logs of the function we will start to see some messages but may need to generate some load before anything appears.
+It may take a few seconds for the application pod to become ready.
+If we take a look at the logs of the application we may start to see some messages.
 
 ```bash
-$ oc logs helloworld-openshift-00001-deployment-5f8dbfb49c-txg8j -c user-container
+oc logs helloworld-openshift-00001-deployment-5f8dbfb49c-txg8j -c user-container
 ```
 
-In order to generate some load we will create a new deployment of the `busybox` image.
+Alternatively, we can generate some Kubernetes events by creating a pod that when starting will generate events such as `pod created` and `container started`. We will create a new deployment of the `busybox` image to show this.
```

Comment:
If there are no events present, we can generate...

We will create a new pod using the `busybox` image to show this.